Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -82,6 +100,12 @@ public class Environment: Environmenting {
 
     /// Sets up the local environment.
     public func bootstrap() throws {
+        // load `.env` variables into `ProcessInfo.processInfo`
+        do {
+            try loadEnvironmentVariablesFromFile()
```

Comment:
I don't think we should blindly load the `.env` file, what about allowing to configure the path in `Config.swift`?
Moreover, if it's not configurable, I wouldn't print anything if it's not present as it would be the majority of the use cases